.github/workflows/kubernetes-install.yaml

```diff
@@ -0,0 +1,112 @@
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Kind
+        run: |
+          curl -Lo ./kind "https://github.com/kubernetes-sigs/kind/releases/download/v0.20.0/kind-$(uname)-amd64"
+          chmod +x ./kind
+          sudo mv kind /usr/local/bin
+          which kind
+      - name: Create Kind cluster
+        run: |
+          cat > /tmp/kind.yaml <<EOF
+          apiVersion: kind.x-k8s.io/v1alpha4
+          kind: Cluster
+          nodes:
+          - role: control-plane
+            image: kindest/node:v1.27.3
+            extraPortMappings:
+            - containerPort: 80
+              hostPort: 80
+          EOF
+          kind create cluster --config /tmp/kind.yaml --wait 5m
+      - name: Build and load gateway
+        run: |
+          docker build -t gateway:test --build-arg TARGETARCH="amd64" -f ./gateway/Dockerfile .
+          kind load docker-image gateway:test
+          docker image rm gateway:test
+      - name: Build and load ray node
+        run: |
+          docker build -t ray:test --build-arg TARGETARCH="amd64" -f ./Dockerfile-ray-node .
+          kind load docker-image ray:test
+          docker image rm ray:test
+      - name: Install helm chart
+        run: |
+          cd charts/quantum-serverless
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+          helm repo add kuberay https://ray-project.github.io/kuberay-helm
+          helm dependency build
+          helm install qs \
+            --set platform=kind \
+            --set jupyterEnable=false \
+            --set repositoryEnable=false \
+            --set prometheusEnable=false \
+            --set lokiEnable=false \
+            --set promtailEnable=false \
+            --set nginxIngressControllerEnable=false \
+            --set gateway.image.repository=gateway \
+            --set gateway.image.tag=test \
+            --set gateway.application.ray.nodeImage=ray:test \
+            --set gateway.application.limits.keepClusterOnComplete=true \
+            --set gateway.secrets.servicePsql.value.databaseName=gateway_db \
+            --set gateway.secrets.servicePsql.value.databasePassword=CHANGE_ME \
+            --set gateway.secrets.servicePsql.value.host=postgresql \
+            --set gateway.secrets.servicePsql.value.port=5432 \
+            --set gateway.secrets.servicePsql.value.userName=bn_keycloak \
+            --set gateway.secrets.superuser.create=true \
+            --set keycloak.postgresql.auth.password=CHANGE_ME \
+            .
+      - name: Give containers time to initialize
+        shell: bash
+        run: sleep 100
+      - name: setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          cd client
+          pip install . --no-cache-dir
+          pip install --no-cache-dir \
+            ipywidgets==8.1.0 \
```

Comment:
It's copied from the notebook dockerfile: https://github.com/Qiskit-Extensions/quantum-serverless/blob/fed9a12dc5a3e3ee84fb33c1d1924fc45414e1c5/Dockerfile-notebook#L21-L29

But it would probably make sense to pull it into a `requirements-notebook.txt` and then use that both places... I'll add in a follow-up PR